Geometry/MTDCommonData/test/runTest.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+function die { echo $1: status $2 ; exit $2; }
+function checkDiff {
+    FSIZE=$(stat -c%s "$1")
+    echo "The output diff is $FSIZE:"
+    cat $1;
+    if [ $FSIZE -gt 7455 ]
+    then
+      exit -1;
+    fi
+}
+
+F1=${CMSSW_BASE}/src/Geometry/MTDCommonData/test/testMTDinDDD.py
+F2=${CMSSW_BASE}/src/Geometry/MTDCommonData/test/testMTDinDD4hep.py
+REF=${CMSSW_BASE}/src/Geometry/TestReference/data/mtdCommonDataRef.log.gz
+FILE1=mtdCommonDataDDD.log
+FILE2=mtdCommonDataDD4hep.log
+LOG=mtdcdlog
+DIF=mtdcddif
+
+echo " testing Geometry/MTDCommonData"
+
+export tmpdir=${LOCAL_TMP_DIR:-/tmp}
+echo "===== Test \"cmsRun testMTDinDDD.py\" ===="
+rm -f $LOG $DIF $FILE1
+(cmsRun $F1 >& $LOG;
+    gzip -f $FILE1; zdiff $FILE1.gz $REF >& $DIF;
+    [ -s $DIF ] && checkDiff $DIF || echo "OK") || die "Failure using cmsRun $F1" $?
```

Comment:
please make sure that script fails in case cmsRun, zdiff or checlDiff fail

Reply:
yes, as I was mentioning earlier, just some care is needed with zdiff as a difference may appears but gives status 1